File: MORM.Java/src/MORM/Java/Models/Venda/Transacao.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "TRANSACAO")
public class Transacao extends CollectionItem
{
    @CampoAttribute(Campo = "ID_TRANSACAO", Tipo = CampoTipo.Key)
    public String Id_Transacao;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "ID_EMPRESA", Tipo = CampoTipo.Req)
    public Integer Id_Empresa;
    @CampoAttribute(Campo = "ID_PESSOA", Tipo = CampoTipo.Req)
    public String Id_Pessoa;
    @CampoAttribute(Campo = "ID_OPERACAO", Tipo = CampoTipo.Req)
    public String Id_Operacao;
    @CampoAttribute(Campo = "DT_TRANSACAO", Tipo = CampoTipo.Req)
    public Date Dt_Transacao;
    @CampoAttribute(Campo = "NR_TRANSACAO", Tipo = CampoTipo.Req)
    public Integer Nr_Transacao;
    @CampoAttribute(Campo = "TP_SITUACAO", Tipo = CampoTipo.Req)
    public Integer Tp_Situacao;
    @CampoAttribute(Campo = "DT_CANCELAMENTO", Tipo = CampoTipo.Nul)
    public Date Dt_Cancelamento;
}

File: MORM.CSharp/MORM.Dominio/Models/Venda/Transacao.cs
using MORM.Util.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("TRANSACAO")]
    public class Transacao : CollectionItem
    {
        [Campo("ID_TRANSACAO", CampoTipo.Key)]
        public string Id_Transacao { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("ID_EMPRESA", CampoTipo.Req)]
        public int Id_Empresa { get; set; }
        [Campo("ID_PESSOA", CampoTipo.Req)]
        public string Id_Pessoa { get; set; }
        [Campo("ID_OPERACAO", CampoTipo.Req)]
        public string Id_Operacao { get; set; }
        [Campo("DT_TRANSACAO", CampoTipo.Req)]
        public DateTime Dt_Transacao { get; set; }
        [Campo("NR_TRANSACAO", CampoTipo.Req)]
        public int Nr_Transacao { get; set; }
        [Campo("TP_SITUACAO", CampoTipo.Req)]
        public int Tp_Situacao { get; set; }
        [Campo("DT_CANCELAMENTO", CampoTipo.Nul)]
        public DateTime Dt_Cancelamento { get; set; }
    }
}